pkg/updater/fetcher.go

```diff
@@ -0,0 +1,61 @@
+package updater
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/httputil"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+const (
+	ifModifiedSinceHeader = "If-Modified-Since"
+	defaultTimeout        = 5 * time.Minute
+)
+
+func newHTTPClient(transport http.RoundTripper) *http.Client {
+	return &http.Client{
+		Timeout:   defaultTimeout,
+		Transport: transport,
+	}
+}
+
+func fetchDumpFromURL(ctx concurrency.Waitable, client *http.Client, url string, lastUpdatedTime time.Time, outputPath string) (bool, error) {
```

Comment:
assuming this is completely copy/pasted, right? I don't think there are any changes here?

Reply:
Correct.